scopesim_templates/micado/spectral_calibrations.py

```
@@ -65,12 +65,24 @@ def line_list(unit_flux=1*PHOTLAM,
     if not isinstance(unit_flux, u.Quantity):
         unit_flux *= PHOTLAM
 
-    # import line list and pad with zeros
-    wave, flux = import_line_spectrum(filename, dwave)
+    do_smoothing = False
+    ksize = 2
+    sigma = 1
 
     if smoothing_fwhm is not None and isinstance(smoothing_fwhm, (int, float)):
```

Do we actually need the `isinstance(smoothing_fwhm, (int, float))` check? The only case where that should apply is if `smoothing_fwhm is None`, which is already checked anyway. If `smoothing_fwhm` has any other type that's incompatible with the rest of the logic, I think it would be better to let the corresponding exception happen instead of silently ignoring it and skipping the smoothing...